examples/rag.py

```diff
@@ -0,0 +1,68 @@
+#  Copyright (c) "Neo4j"
```

Comment:
It's minor but we discussed not having the copyright header for examples here

https://github.com/neo4j/neo4j-genai-python/pull/52#discussion_r1626132952